templates/post-edit.html

```diff
@@ -53,8 +53,8 @@
 </div>
 <script>
     document.addEventListener("DOMContentLoaded",() => {
-        document.getElementById('imagefile').addEventListener('change', (event) => {
-            image = document.getElementById('imagefile')
+        document.getElementById('post-file').addEventListener('change', (event) => {
+            image = document.getElementById('post-file')
```

Comment:
Why not "post-image"?

Reply:
because another element is post-image

![image](https://user-images.githubusercontent.com/112652150/229364981-e94e8840-6bd2-4159-ad1d-d269925095b7.png)